libs/server-sdk/src/client_impl.cpp

```diff
@@ -177,24 +196,31 @@ AllFlagsState ClientImpl::AllFlagsState(Context const& context,
     std::unordered_map<Client::FlagKey, Value> result;
 
     if (!Initialized()) {
-        if (memory_store_.Initialized()) {
-            LD_LOG(logger_, LogLevel::kWarn)
-                << "AllFlagsState() called before client has finished "
-                   "initializing; using last known values from data store";
-        } else {
-            LD_LOG(logger_, LogLevel::kWarn)
-                << "AllFlagsState() called before client has finished "
-                   "initializing. Data store not available. Returning empty "
-                   "state";
-            return {};
-        }
+        //        if (memory_store_.Initialized()) {
```

Comment:
This is commented out because I haven't yet implement the initialization states in the DS spec.